### PR TITLE
Fix CC capacity config example typos

### PR DESCRIPTION
**Pull request by tomncooper**
### Type of change

Bugfix

### Description

Fixes a typo in the Cruise Control deployment guides broker capacity config.

### Checklist

- [X] Update documentation

### CODE REVIEW

**Comment by SpeedEX** [comment]
I think there are more typo in the doc that related to capacity configuration.
1. In this file also https://github.com/strimzi/strimzi-kafka-operator/blob/master/documentation/modules/cruise-control/con-optimization-goals.adoc
2. The `networkIn` and `networkOut` key is also not correct, I think the corrected one should be `inboundNetwork` and `outboundNetwork` according to Spec https://github.com/strimzi/strimzi-kafka-operator/blob/master/documentation/modules/cruise-control/ref-cruise-control-configuration.adoc#capacity-configuration

**Comment by tomncooper** [comment]
You are correct again! I have fixed those and will go over the all the CC capacity configs again. Thanks again for your input.

**Comment by tomncooper** [comment]
@laidan6000 I think I have corrected all the capacity config keys in the example yaml, but our docs structure is not the most easy to navigate. Can you have a look over these as well and make sure the capacity configs are correct.

File: documentation/modules/cruise-control/con-optimization-goals.adoc [hunk 1]
```diff
@@ -81,8 +81,8 @@ spec:
     userOperator: {}
   cruiseControl:
     capacity:
```
Comment (SpeedEX) [inline documentation/modules/cruise-control/con-optimization-goals.adoc:83]: I think this line should also be `brokerCapacity:`

File: documentation/modules/cruise-control/con-optimization-goals.adoc [hunk 2]
```diff
@@ -154,8 +154,8 @@ spec:
...
```
Comment (SpeedEX) [inline documentation/modules/cruise-control/con-optimization-goals.adoc:156]: I think this line should also be `brokerCapacity:`

**Comment by tomncooper** [comment]
Ok (thanks to @SpeedEX) I think I have them all now!

**Comment by SpeedEX** [comment]
Just my curiosity, how about the document in the release 0.18 branch ?

**Comment by tomncooper** [comment]
@scholzj will this need backporting to the 0.18 branch to update the website docs?

**Comment by scholzj** [comment]
I'm not sure how easy it would be to backport - depends on how much the docs changed since the release. We do not do that in general.

**Comment by SpeedEX** [comment]
> I'm not sure how easy it would be to backport - depends on how much the docs changed since the release. We do not do that in general.

Just my curiosity (again), what the actual step of backport the document ? can't we just create another PR for that branch ?

**Comment by scholzj** [comment]
> Just my curiosity (again), what the actual steps of backport the document ? can't we just create another PR for that branch ?

The documentation is build from the release tag. But the release tag already exists and should not change for a release which is several weeks old. I cannot just tag the commit from this PR. So it would include some manual work to get the the _right_ sources but also to manually build and push the docs.

Also, there were lot of other changes to the documentation since the release. So the changes might not be trivial to backport (I haven't tried - so I do not know).

**Comment by SpeedEX** [comment]
So I will leave the back-porting decision to you all then. I just fear that someone might face the same confusion as me when read the 0.18 document. 

Thanks for the quick respond, love this project.

**Comment by laidan6000** [comment]
@tomncooper - I spotted one more potential reference to `capacity` instead of `brokerCapacity` in the Cruise Control chapter.

See https://strimzi.io/docs/operators/master/using.html#ref-cruise-control-configuration-str

The restricted options include `capacity.config.file`.

I can backport these changes to the 0.18.0 release at some point soon.

**Comment by tomncooper** [comment]
> The restricted options include `capacity.config.file`.

That is an actual CC config option, the `brokerConfig` comes from Strimzi.

**Pull request merged**